WORKSPACE

```diff
@@ -5113,24 +5000,6 @@ rpm(
     ],
 )
 
-rpm(
-    name = "openldap-0__2.6.6-3.el9.aarch64",
-    sha256 = "3553d2a8ef3901444aae018a0f1a17f41d3ffba77af79735e8422c37f6ac57d8",
-    urls = [
```

Comment:
Strange that these rpms are just removed for arm64 & s390x but not x86?

Reply:
Let me look into it.

Reply:
Okay, apparently that was dragged in by one of the packages in `launcherbase` (not sure which one) and that's no longer the case, so for aarch64 and s390x it's just gone.

On x86_64 however, we have the `libguestfstools` container too, which is roughly a superset of `launcherbase`, and in that case some package apparently still requires openldap. So in that case it can't be dropped.

There are a number of other packages that we only need on x86_64 for the same reason: examples include hexedit, libosinfo, augeas-libs, fuse.

Overall I think we're good.

Reply:
Thanks for looking into it!